config/peerpods/podvm/aws-VM-image-delete-job.yaml

```diff
@@ -26,13 +26,13 @@ spec:
         env:
           - name: PODVM_DISTRO
             value: rhel
-          - name: IMAGE_NAME
-            value: "peer-pod-ami"
         command:
         - /bin/sh
         - -c
         - |
           set -e
+          [[ ! "${PODVM_AMI_ID}" ]] && echo "PODVM_AMI_ID is missing, it's unknown which image to delete" && exit 1
+          [[ "${IMAGE_NAME}" ]] && echo "IMAGE_NAME:${IMAGE_NAME} is set, it implies image was not automatically created, delete it manually" && exit 0
```

Comment:
I think we need to make sure it is clear in docs as to what happens with manually created images and that user is responsible for deleting them as the message states.

Reply:
Is custom images supported ATM? AFAIK the reason it's supported is for tech/dev-preview features only